core-web/apps/dotcms-ui/src/app/portlets/dot-apps/dot-apps-configuration-detail/dot-apps-configuration-detail-form/dot-apps-configuration-detail-form.component.html

```diff
@@ -1,25 +1,19 @@
-<ng-template #warningIcon let-field="field">
-    <dot-icon
-        *ngIf="field.warnings && field.warnings.length"
-        name="warning"
-        pTooltip="{{ field.warnings.length ? field.warnings.join('. ') : '' }}"
-        size="18"
-    ></dot-icon>
-</ng-template>
+<form class="p-fluid" #form="ngForm" [formGroup]="myFormGroup" novalidate>
+    <ng-template #warningIcon let-field="field">
+        <dot-icon
+            *ngIf="field.warnings && field.warnings.length"
+            name="warning"
+            pTooltip="{{ field.warnings.length ? field.warnings.join('. ') : '' }}"
+            size="18"
+        ></dot-icon>
+    </ng-template>
 
-<ng-template #labelField let-field="field">
-    <label
-        [for]="field.name"
-        class="form__label"
-        [ngClass]="{
-            required: field.required
-        }"
-        >{{ field.label }}</label
-    >
-</ng-template>
-
-<form #form="ngForm" [formGroup]="myFormGroup" class="p-fluid" novalidate>
-    <div #formContainer class="dot-apps-configuration-detail__form">
+    <ng-template #labelField let-field="field">
+        <label [for]="field.name" [checkIsRequiredControl]="field.name" dotFieldRequired>{{
```

Comment:
This Input not exist in dotFieldRequired anymore.

Reply:
Or is the for of the label?

Reply:
it is for field and label not for directive.